API.md

```diff
@@ -424,6 +424,43 @@ parser.parse('accccc');
 //=> {status: true, value: ['a', ['c', 'c', 'c', 'c', 'c']]}
 ```
 
+# Binary constructors.
+
+The purpose of the following constructors is to allow the consumption of Buffer types in node to allow for attoparsec style consumption of binary input.
+As these constructors yield regular values within parsers, they can then be combined in the same fashion as the above string-based constructors to produce
+robust binary parsers.  These constructors live in the Parsimmon.Binary namespace.
+
+## Parsimmon.byte(int)
+
+Returns a parser that yields a byte that matches the given input.  Similar to digit/letter.
+
+## Parsimmon.bitSeq(...alignments)
```

Comment:
I think you should write `[...alignments]` or just `alignments` here since the current syntax style makes me immediately think you'd call it like `bitSeq(1, 2, 5)` not `bitSeq([1, 2, 5])`

Reply:
that's a good point, do you think I should make it based off arguments rather than an array?

Reply:
Well, I actually really don't like using `arguments` and would eventually like to change the other functions to use explicit arrays instead (I just haven't because I don't think I can do this without a breaking API change; maybe detect if there's exactly one param and it's an array? could mess up `seqObj` though I think—), so personally I'm cool with arrays here